nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/avro/Constants.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.ecm.core.io.avro;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Avro local constants.
+ *
+ * @since 10.2
+ */
+class Constants {
+
+    public static final String ACL = "acl";
+
+    public static final String CHANGE_TOKEN = "changeToken";
+
+    public static final String CONTENT = "content";
+
+    public static final String CURRENT_LIFE_CYCLE_STATE = "currentLifeCycleState";
+
+    public static final String DATA = "data";
+
+    public static final String DIGEST = "digest";
+
+    public static final String DOCUMENT_MODEL = "documentModel";
+
+    public static final String DOCUMENT_TYPE = "documentType";
+
+    public static final String ENCODING = "encoding";
+
+    public static final String IS_CHECKIN = "isCheckedIn";
+
+    public static final String IS_LATEST_MAJOR_VERSION = "isLatestMajorVersion";
+
+    public static final String IS_LATEST_VERSION = "isLatestVersion";
+
+    public static final String IS_PROXY = "isProxy";
+
+    public static final String IS_TRASHED = "isTrashed";
+
+    public static final String IS_VERSION = "isVersion";
+
+    public static final String MIME_TYPE = "mime-type";
+
+    public static final String MIXIN_TYPES = "mixinType";
+
+    public static final String NAME = "name";
+
+    public static final String PARENT_ID = "parentId";
+
+    public static final String PATH = "path";
+
+    public static final String POS = "pos";
+
+    public static final String PRIMARY_TYPE = "primaryType";
+
+    public static final String REPOSITORY = "repository";
+
+    public static final String SCHEMA = "schema";
+
+    public static final String TAGS = "tag";
+
+    public static final String TIMESTAMP_MILLIS = "timestamp-millis";
```

Comment:
This seems to be a different type of constant, not an Avro field name. It's worth reflecting that in the constant name (with a prefix or suffix) and grouping them together.

Reply:
done